langchain/vectorstores/chroma.py

```diff
@@ -459,7 +464,12 @@ def persist(self) -> None:
                 "You must specify a persist_directory on"
                 "creation to persist the collection."
             )
-        self._client.persist()
+        import chromadb
+
+        # Maintain backwards compatibility with chromadb < 0.4.0
+        major, minor, _ = chromadb.__version__.split(".")
+        if int(major) >= 0 and int(minor) < 4:
```

Comment:
@gpapp is this supposed to be int(major) == 0?

Reply:
Of course it is! Good spot, I managed to miss that in the rush.

Reply:
good to land this and https://github.com/hwchase17/langchain/pull/7749 once this change lands